### ISSUE TITLE
Share button copy does not contain a link to the game

### ISSUE THREAD

**Issue by stshan** [the-abe-train/globle#72]
per title, every other wordle variant contains a link when you copy your results so it makes it easy to access the website. Globle still requires me to type it into google manually

**Comment by Luxemvia** [comment]
Yes! Please update the ‘Share’ content to include a link back to the game. While it’s not bad, “#globle” isn’t as user friendly. Perhaps just make the # tag the link?

**Comment by olliechick** [comment]
Wordle itself, however, does not. The URL was initially removed because Facebook banned it (#53)
